### PR TITLE
fixing spelling errors/grammatically incorrect english

### PR DESCRIPTION
**Pull request by evynhh**

### CODE REVIEW

File: src/js/avatarcropper.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export class AvatarCropper extends Widget {
 
         if (getIEVersion() !== false && !AvatarCropper.settings.dismissedIE) {
             window.alert(
-                "hey so your browser isn't really supported ... things should still work but they will be slower/ugly due to how internet explorer/edge function (They don't conform to web standards). i'd recommend switching to firefox or chrome!! but you don't have to if you don't want to. this is the only time you'll see this message unless u clear ur cache or something. ok bye",
+                "Your browse (IE, presumably) is not supported. Some things may show up incorrectly. Switch to Chrome, Firefox, or modern Edge for a better experience. You'll only see this message once.",
```
Comment (TheOddball) [inline src/js/avatarcropper.ts:129]: Browser*!

File: package-lock.json [hunk 2]
```diff
@@ -1,10355 +1,8 @@
 {
     "name": "avatarcropper",
     "version": "1.0.0",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```
Comment (TheOddball) [inline package-lock.json:4]: Please don't delete package lock :(

**Review by TheOddball — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (evynhh) [reply package-lock.json:4]: i didn't change that myself, something else must've changed it

File: src/js/avatarcropper.ts (hunk 1 above)
Comment (evynhh) [reply src/js/avatarcropper.ts:129]: ah thanks for catching that lol

**Review by TheOddball — APPROVED** [review]

**Comment by evynhh** [comment]
are they dead

**Comment by SongSing** [comment]
this is how i talk so it tends to be how my web apps talk, im not super interested in changing it atm

**Comment by evynhh** [comment]
so i did that for nothing. got it.

**Pull request closed without merging**